pages/common/nmap.md

```diff
@@ -4,6 +4,16 @@
 > Some features only activate when Nmap is run with privileges.
 > More information: <https://nmap.org>.
 
+- If Nmap throws a root privileges error use it with `sudo` or `admin rights cmd`
+
+- To check if the IP address `192.168.1.101` is up, with it’s operating system info. It may show RTTVAR logs (for TCP) for a long time:
+
+`nmap -O 192.168.1.101`
+
+- To scan the Network for devices (including routers) and get info on their operating system:
```

Comment:
Perhaps this will read better?
```suggestion
- Scan a block of IP addresses, and additionally attempt to guess the operating system on any hosts found:
```